src/app/component/login/Login.tsx

```diff
@@ -40,88 +49,134 @@ const Login = () => {
-                  <span className="mb-1 text-xl font-medium text-gray-900 dark:text-white">Address: </span>
-                  <span className="mb-1 text-sm font-medium text-gray-900 dark:text-white" style={{ lineBreak: "anywhere" }}> {ensName ? `${ensName} (${address})` : address}</span>
+                  <span className="mb-1 text-xl font-medium text-gray-900 dark:text-white">
+                    Connected to {connector?.name}
+                  </span>
+                  <span className="mb-1 text-xl font-medium text-gray-900 dark:text-white">
+                    Address:{" "}
+                  </span>
+                  <span
+                    className="mb-1 text-sm font-medium text-gray-900 dark:text-white"
+                    style={{ lineBreak: "anywhere" }}
+                  >
+                    {" "}
+                    {ensName ? `${ensName} (${address})` : address}
+                  </span>
                   <div className="flex mt-4 space-x-3 md:mt-6">
-                    <button onClick={disconnectWallet} className="inline-flex items-center px-4 py-2 text-sm font-medium text-center text-white bg-appColor1 rounded-sm">Disconnect</button>
+                    <button
+                      onClick={disconnectWallet}
+                      className="inline-flex items-center px-4 py-2 text-sm font-medium text-center text-white bg-appColor1 rounded-sm"
+                    >
+                      Disconnect
+                    </button>
                   </div>
                 </div>
               </div>
-              :
-              <div >
+            ) : (
+              <div>
                 {connectors.map((connector) => (
-                  <button key={connector.id} onClick={() => connect({ connector })} className="flex cursor-pointer items-center w-full mb-2 bg-white text-black rounded-lg flex-row  hover:bg-gray-100">
-                    <div className='rounded-l-lg px-3 py-2 bg-appColor1'>
-                      {connector.name === 'MetaMask' && <MetaMask />}
-                      {connector.name === 'Coinbase Wallet' && <CoinBase />}
-                      {connector.name === 'WalletConnect' && <WalletConnect />}
-                      {connector.name === 'Injected' && <WalletConnect />}
+                  <button
+                    key={connector.id}
+                    onClick={() => connect({ connector })}
+                    className="flex cursor-pointer items-center w-full mb-2 bg-white text-black rounded-lg flex-row  hover:bg-gray-100"
+                  >
+                    <div className="rounded-l-lg px-3 py-2 bg-appColor1">
+                      {connector.name === "MetaMask" && <MetaMask />}
+                      {connector.name === "Coinbase Wallet" && <CoinBase />}
+                      {connector.name === "WalletConnect" && <WalletConnect />}
+                      {connector.name === "Injected" && <WalletConnect />}
```

Comment:
This contenct is duplicated in other file down here. 
Can we create something common ? (check how much must be common and how much not)

Reply:
this Login will be removed. its not needed anymore. 

the content will be used one place after the login component is deleted. I will handle that later